monitoring/enabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,42 @@
+[id="enabling-alert-routing-for-user-defined-projects"]
+= Enabling alert routing for user-defined projects
+include::modules/common-attributes.adoc[]
+:context: enabling-alert-routing-for-user-defined-projects
+
+toc::[]
+
+:FeatureName: Alert routing for user-defined projects 
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+In {product-title} {product-version}, a cluster administrator can enable alert routing for user-defined projects. 
+After doing so, the administrator can allow users to configure alert routing for their user-defined projects.
+Users then create and configure user-defined alert routing by editing the `AlertmanagerConfig` custom resource description (CRD).
+After a user has defined alert routing for a user-defined project, user-defined alerts are routed to the `alertmanager-main`` pods in the `openshift-monitoring` namespace. 
```

Comment:
`alertmanager-main``
should be
`alertmanager-main`

Reply:
Fixed